Command/DebugCommand.php

```diff
@@ -22,37 +23,17 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $r = new \ReflectionProperty($router, 'receivers');
         $r->setAccessible(true);
 
-        $maxWidths = array(
-            'name' => 4,
-            'service' => 7,
-        );
-
-        $rows = array();
-
-        foreach ($r->getValue($router) as $name => $id) {
-            $maxWidths['name'] = max(4, strlen($name));
-            $maxWidths['service'] = max(7, strlen($id));
-
-            $rows[] = array($name, $id);
-        }
-
-        $header = '| <info>%-' . $maxWidths['name'] . '.' . $maxWidths['name'] . 's</info> | <info>%-' . $maxWidths['service'] . '.' . $maxWidths['service'] . 's</info> |';
-        $pattern = '| <comment>%-' . $maxWidths['name'] . '.' . $maxWidths['name'] . 's</comment> | <comment>%-' . $maxWidths['service'] . '.' . $maxWidths['service'] . 's</comment> |';
-
-        $this->writeLineDelimiter($output, $maxWidths);
-
-        $output->writeln(sprintf($header, 'Message', 'Service'));
-
-        $this->writeLineDelimiter($output, $maxWidths);
-
-        foreach ($rows as $row) {
-            $output->writeln(sprintf($pattern, $row[0], $row[1]));
-            $this->writeLineDelimiter($output, $maxWidths);
+        $rows = [];
+        foreach ($r->getValue($router) as $key => $val) {
+            $rows[] = [$key, $val];
         }
-    }
 
-    protected function writeLineDelimiter($output, $maxWidths)
-    {
-        $output->writeln('+' . str_repeat('-', $maxWidths['name'] + 2) . '+' . str_repeat('-', $maxWidths['service'] + 2) . '+');
+        /** @var \Symfony\Component\Console\Helper\TableHelper $helper */
```

Comment:
Can you elaborate? This is not only about the IDE autocomplete feature. It defines what type the return type supposed to be. It precises the helper type instead of relying on `Symfony\Component\Console\Helper\HelperInterface`. I think of it as Java's cast e.g. $helper = (TableHelper) $this->getHelper('table');

It is not used wrong by my editor. I put in there manually on purpose. Do you mean `$helper` should be placed right after the `@var`?